app-text/mandoc/mandoc-1.14.3.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="A suite of tools compiling mdoc from the OpenBSD project"
+DESCRIPTION="A suite of tools compiling mdoc, the roff macro language of choice \
+for BSD manual pages, and man, the predominant historical language for UNIX \
+manuals. It is small, ISO C, ISC-licensed, and quite fast. The main component \
+of the toolset is the mandoc utility program, based on the libmandoc validating \
+compiler, to format output for UTF-8 and ASCII UNIX terminals, HTML 5, \
+PostScript, and PDF."
+HOMEPAGE="http://mandoc.bsd.lv/"
+COPYRIGHT="1989-2017 mandoc Authors"
+# Note: update LICENSE on every new version
+LICENSE="mandoc-$portVersion"
+REVISION="1"
+SOURCE_URI="http://mandoc.bsd.lv/snapshots/mandoc-$portVersion.tar.gz"
+CHECKSUM_SHA256="0b0c8f67958c1569ead4b690680c337984b879dfd2ad4648d96924332fd99528"
+PATCHES="mandoc-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	mandoc$secondaryArchSuffix = $portVersion
+	cmd:apropos = $portVersion
+	cmd:demandoc = $portVersion
+	cmd:makewhatis = $portVersion
+	cmd:man = $portVersion
+	cmd:mandoc = $portVersion
+	cmd:mandocd = $portVersion
+	cmd:soelim = $portVersion
+	cmd:whatis = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+CONFLICTS="
+	cmd:man
+"
```

Comment:
```diff
 CONFLICTS="
 	cmd:man
-"
+	"
```

Reply:
Any tips regarding commiting whitespace fixes? git doesn't register this one for me :(